src/types/Wallet/methods/exportWallet.js

```diff
@@ -5,11 +5,31 @@ function exportMnemonic(mnemonic) {
   return mnemonic.toString();
 }
 
+function exportPublicKeyWallet(_outputType = 'publicKey') {
```

Comment:
Please don't use underscore in variable names

Reply:
Indeed! I just pushed a commit to fix style on the whole exportWallet file !